cumulusci.yml

```diff
@@ -51,3 +53,22 @@ flows:
         steps:
             3:
                 task: load_storytelling_data
+
+    config_qa:
+        steps:
+            3:
+                task: load_storytelling_data
+
+    config_managed:
+        steps:
+            3:
+                task: load_storytelling_data
+
+    2gp_org:
```

Comment:
We probably need to ship this flow or something very similar in the standard library.

Reply:
🥳